charts/mock-smtp/values.yaml

```diff
@@ -77,8 +77,8 @@ services:
 
 image:
   registry: docker.io
-  repository: mosipqa/mock-smtp
-  tag: 0.0.2
+  repository: mosipid/mock-smtp
+  tag: 0.0.1
```

Comment:
0.0.2

Reply:
mock smtp not released sir

Reply:
release mock smtp 0.0.2 and update relevant changes